packages/app/src/components/LoadFlameModal/LoadFlameModal.tsx

```diff
@@ -29,11 +25,11 @@ function Preview(props: { flameDescriptor: FlameDescriptor }) {
           position={vec2f(
             ...props.flameDescriptor.renderSettings.camera.position,
           )}
-          fovy={zoomToFovy(props.flameDescriptor.renderSettings.camera.zoom)}
+          zoom={props.flameDescriptor.renderSettings.camera.zoom}
         >
           <Flam3
-            quality={0.8}
-            pointCountPerBatch={2e4}
+            quality={0.95}
+            pointCountPerBatch={1e5}
```

Comment:
this is quite high for my PC. This means that 1e5 * example count (9) points, almost 1M are running when example dialog is opened. It will definitely lag on weaker machines.

Reply:
Ah yes, I was just testing this, I think we could add another set of env variables that I can set for myself, because the quality of those is a bit weak for some flames, would be perhaps also good to set it per example, so we can adjust more?

Reply:
I'll do this in next feature, where I add proper variation gallery, then see how can I optimize examples and load times